modules/internal/Bytes.chpl

```diff
@@ -963,25 +963,63 @@ module Bytes {
 
     /*
       Returns a UTF-8 string from the given :record:`bytes`. If the data is
-      malformed for UTF-8, `errors` argument determines the action.
+      malformed for UTF-8, `policy` argument determines the action.
       
-      :arg errors: `decodePolicy.strict` raises an error, `decodePolicy.replace`
+      :arg policy: `decodePolicy.strict` raises an error, `decodePolicy.replace`
                    replaces the malformed character with UTF-8 replacement
                    character, `decodePolicy.ignore` drops the data silently,
```

Comment:
If we're deprecating `decodePolicy.ignore`, should we adjust the docs here to change mentions of `decodePolicy.ignore` to `decodePolicy.drop`?